docs/specs/moniker.yml

```diff
@@ -1595,3 +1595,76 @@ outputs:
     { "monikers": ["netcore-1.0"] }
   4667fedf/a.json: |
     { "monikers": ["netcore-2.0"] }
+---
+# Allow monikers Config if monikerRange not defined
+inputs:
+  docfx.yml: |
+    monikerRange:
+      'docs/v1/**': '>= netcore-2.0'
+    routes:
+      docs/v1/: docs/
+    monikerDefinition: monikerDefinition.json
+  docs/v1/TOC.md: |
+    ---
+    monikers: netcore-2.0
+    ---
+    # Node
+  monikerDefinition.json: |
+    {
+      "monikers": [
+        { "moniker_name": "netcore-1.0", "product_name": ".NET Core" },
+        { "moniker_name": "netcore-2.0", "product_name": ".NET Core" },
+      ]
+    }
+outputs:
+  4667fedf/docs/toc.json:
```

Comment:
>4667fedf/docs/toc.json: [](start = 2, length = 23)

add the detail moniker list make the expected output more intuitive

Reply:
sure